### DISCUSSION TITLE
Projects (beta) - allow grouping by "GitHub data" like assignee

### DISCUSSION THREAD

**Discussion by felixfbecker** [community/community#5448] [Projects and Issues] [answered]
The [docs](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/customizing-your-project-views#group) say:

> In table layout, you can group items by a custom field value. **You cannot group by fields that contain GitHub data (such as: title, assignees, repository, labels, milestone).**

However, for iteration planning, I always would like to group my table by **assignee** so work can be distributed equally for an iteration.

This would be especially important when https://github.com/github/feedback/discussions/5376 is made possible, so that summing up per assignee (team member) is possible to ensure work is distributed equally and nobody has more work on their plate than is time in the iteration.

~~Grouping by **milestone** would also be very helpful for a longer-term planning view.~~ This is already possible, the docs are just incorrect and it can't be accessed through the command palette.

**Comment by CodeCox** [comment]
I, fortunately, did not read the doc closely so was able to group by _Milestone_.

![image](https://user-images.githubusercontent.com/44036684/132109499-4715a3cb-5046-4ac8-8109-6d41093ee739.png)

But I just checked and sadly it seems that it is the only field from that list that can be grouped.

**Reply by chrisreddington** [reply]
I was hoping to be able to group by Labels. I use labels to categorise some of my issues into various content buckets. I suppose I could add a custom field for this purpose, but then it feels like duplication.

**Reply by Totktonada** [reply]
I would like to group issues per repository. Now, it seems, there is no such ability.

![image](https://user-images.githubusercontent.com/92749/132534215-899d3409-9396-429f-9808-0fd2f0db96ac.png)

Update: Found the relevant [thread](https://github.com/github/feedback/discussions/5217).

**Comment by CodeCox** [comment]
This section  of the [docs](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/customizing-your-project-views#save-views) suggests that _group by Assignee_ is possible!

___
![image](https://user-images.githubusercontent.com/44036684/132674801-7ca07e74-c66b-416b-bd21-a8e03689b0f4.png)
___

Perhaps this use-case is a work-in-progress or on the roadmap?

**Comment by GooBall** [comment]
For me, grouping by Assignee is one of the main "missing" features. I hope it is as @CodeCox says and it is just WIP.

**Comment by bvaughn** [comment]
Grouping by Assignee(s) would provide a way to quickly tell if an individual was overloaded for a given milestone/project. Would be very helpful!

Only ambiguous part of the feature (and perhaps why it's disabled currently?) would be: How to group an issue with multiple assignees? Do you show the union as its own group, or do you show the issue multiple times (one per each assignee)? I think I'd prefer the latter.

**Comment by Totktonada** [comment, marked as answer]
It seems, it is tracked in the https://github.com/github/roadmap/issues/283 roadmap issue now:

> - [ ] Group-by assignee and repos

It is in the Q4 2021 milestone ATM.
